addons/crm/models/crm_lead.py

```diff
@@ -350,6 +350,17 @@ def _compute_name(self):
             if not lead.name and lead.partner_id and lead.partner_id.name:
                 lead.name = _("%s's opportunity") % lead.partner_id.name
 
+    @api.depends('partner_id')
+    def _compute_lang_id(self):
```

Comment:
Linting: would define it just above "_compute_partner_address_values" (to try to keep ordering somehow the same between fields definition and compute methods definition)